src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -179,6 +179,9 @@ def _load_refcase(refcase_file: Optional[str]) -> Optional[EclSum]:
         if refcase_file is None:
             return None
 
+        refcase_filepath = Path(refcase_file)
+        refcase_file = str(refcase_filepath.parent / refcase_filepath.stem)
+
         if not os.path.exists(refcase_file + ".UNSMRY"):
```

Comment:
Outside the scope here, so no need to fix it, but since you are now using pathlib you can do:
```python
if not (refcase_file / ".UNSMRY").exists:
```